appveyor.yml

```diff
@@ -4,17 +4,14 @@
 # https://github.com/rmcgibbo/python-appveyor-conda-example
 
 
-matrix:
-  fast_finish: true     # immediately finish build once one of the jobs fails.
-
 environment:
   matrix:
-    - PYTHON_VERSION: "2.7"
-      MINICONDA: C:\Miniconda-x64
-
     - PYTHON_VERSION: "3.5"
       MINICONDA: C:\Miniconda35-x64
 
+    - PYTHON_VERSION: "2.7"
```

Comment:
I reverted b/c even after I changed above appveyor refused to finish the second job. I guess that is b/c of how appveyor read the commits or something...